macosx/HBTitle.m

```diff
@@ -25,12 +25,37 @@
 extern NSString *keySubTrackLanguageIsoCode;
 extern NSString *keySubTrackType;
 
+@interface HBMetadata ()
+
+@property (nonatomic, readonly) hb_metadata_t *hb_metadata;
+@property (nonatomic, readwrite, copy) NSString *releaseDate;
```

Comment:
If releaseDate is going to be readonly there is no need to redeclare it as readwrite here.

Reply:
- [ ] fix rw assignment on releaseDate property